client/main.lua

```diff
@@ -0,0 +1,10 @@
+local config = require 'config.client'
+
+function DropFingerprint()
+    if IsWearingGloves() then return end
+
+    local coords = GetEntityCoords(cache.ped)
```

Comment:
move the coords var inside the if
https://manason.github.io/effective-fivem-lua/variables/#location